pyglet/app/base.py

```diff
@@ -121,6 +122,9 @@ def stop(self):
         pass
 
 
+@register_event_type('on_window_close')
```

Comment:
Just a cosmetic nitpick, but it looks a little more aesthetically pleasing if the decorators are stacked from smallest to largest/

Reply:
done